python/cudf/cudf/core/column/string.py

```diff
@@ -5226,6 +5226,55 @@ def edit_distance_matrix(self) -> SeriesOrIndex:
             libstrings.edit_distance_matrix(self._column)
         )
 
+    def minhash(
+        self,
+        seeds: Optional[cudf.Series] = None,
+        n: int = 4,
+        method: str = "murmur3",
+    ) -> SeriesOrIndex:
+        """
+        Compute the minhash of a strings column.
+
+        Parameters
+        ----------
+        seeds : Series
+            The seeds used for the hash algorithm.
+            Must be of type uint32.
+        n : int
+            The width of the substring to hash.
+            Default is 4 characters.
+        method : str
+            Hash function to use.
+            Only 'murmur3' (MurmurHash3_32) is supported.
+            Default is 'murmur3'.
+
+        Examples
+        --------
+        >>> import cudf
+        >>> str_series = cudf.Series(['this is my', 'favorite book'])
+        >>> seeds = cudf.Series([0], dtype=np.uint32)
+        >>> str_series.str.minhash(seeds)
+        0     21141582
```

Comment:
I must be missing something -- is the list unwrapped into a numerical column if there is only one seed? Is that being done in C++ or Python? Is that documented? I glanced over the implementation again and didn't see it.

Reply:
This happens in the libcudf API. I will update the doxygen.